frontend-react/src/components/ManagePublicKey/ManagePublicKeyUpload.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useRef } from "react";
+import {
+    Button,
+    FileInput,
+    FileInputRef,
+    Form,
+    FormGroup,
+    Label,
+} from "@trussworks/react-uswds";
+
+export interface ManagePublicKeyFormProps {
```

Comment:
It looks like we're vacillating between types and interfaces for props.  Let's choose one (I vote interfaces for props) and stick with that structure.